programs/server/Server.cpp

```diff
@@ -239,10 +239,7 @@ int waitServersToFinish(std::vector<DB::ProtocolServerAdapter> & servers, size_t
         current_connections = 0;
 
         for (auto & server : servers)
-        {
-            server.stop();
```

Comment:
I know this line is going to be executed after `stop()` is already called for each server, but it seems there is no harm to call `stop()` again and by the logic of this function it's probably better to keep `stop()` here.

Reply:
Indeed, it is redundant, and for me it does not looks like waitServersToFinish should stop the server too. But I don't have any strong feeling about this, so OK, I will get back it.